enhancements/cluster-logging/gcp-external-account-auth.md

```diff
@@ -0,0 +1,201 @@
+---
+title: gcp-external-account-auth
+authors:
+  - "@cahartma"
+reviewers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@Clee2691"
+  - "@JoaoBraveCoding"
+  - "@xperimental"
+approvers:
+  - "@jcantrill"
+  - "@alanconway"
+api-approvers: 
+  - "@jcantrill"
+creation-date: 2025-02-27
+last-updated: 2024-02-27
+tracking-link:
+  - "https://issues.redhat.com/browse/LOG-3577"
+see-also:
+  - "https://issues.redhat.com/browse/LOG-3275"
+  - "https://issues.redhat.com/browse/LOG-6762"
+replaces: []
+superseded-by: []
+---
+
+# GCP Workload Identity Federation (WIF) Support in OpenShift Logging
+
+This enhancement enables secure, short-lived authentication for GCP services in OpenShift Logging using Workload Identity Federation
```

Comment:
This enhancement enables secure, short-lived authentication for forwarding logs to GCP services in OpenShift Logging using Workload Identity Federation